This is a testharness.js-based test.
[FAIL] Service Worker responds to fetch event with the referrer policy
  promise_test: Unhandled rejection with value: object "TypeError: Failed to fetch"
Harness: the test ran to completion.
